api/acl/management/commands/check_acl.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+from acl.models import Acl
+
+
+class Command(BaseCommand):
+    help = 'Check if an ACL exists'
+
+    def add_arguments(self, parser):
+        parser.add_argument('service_account', help='Service account')
+        parser.add_argument('zone', help='Zone')
+
+    def handle(self, *args, **options):
+        acl = Acl.objects.filter(service_account=options['service_account'], zone=options['zone']).first()
```

Comment:
As another alternative get() is more explicit that there is only one.

Reply:
Done